### PR TITLE
[DependencyInjection] Document `stack` definitions

### LINKED ISSUE #13587: [DI] add syntax to stack decorators
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#36373
| PR author(s) | @nicolas-grekas
| Merged in    | 5.1-dev

### PR DESCRIPTION
**Pull request by HeahDude**
Fixes #13587.

### CODE REVIEW

File: service_container/service_decoration.rst [hunk 1]
```diff
@@ -316,6 +317,226 @@ The generated code will be the following::
...
+                'first' => inline_service()->parent('embedded_stack'),
+                'second' => inline_service(\Baz::class),
+                // ...
+            ])
+
+    The ``Baz`` frame id will now be ``.decorated_foo_stack.second``.
+
+.. versionadded:: 5.1
+
+    Defining ``stack`` was introduced in Symfony 5.1.
```
Comment (OskarStark) [inline service_container/service_decoration.rst]:
```suggestion
    The ability to define a ``stack`` was introduced in Symfony 5.1.
```
?

**Review by OskarStark — APPROVED** [review]
Great work!

File: service_container/service_decoration.rst (hunk 1 above)
Comment (HeahDude) [reply service_container/service_decoration.rst]: Done, thanks!

**Review by nicolas-grekas — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you Jules.

**Pull request merged**